static/app/components/modals/inviteMembersModal/inviteRowControl.tsx

```diff
@@ -115,8 +114,6 @@ class InviteRowControl extends React.Component<Props, State> {
           styles={getStyles(theme, inviteStatus)}
           onInputChange={this.handleInputChange}
           onKeyDown={this.handleKeyDown}
-          onBlurResetsInput={false}
-          onCloseResetsInput={false}
```

Comment:
Do these properties no longer work?

Reply:
These are from react-select 1.x

They weren't a type error because of this https://github.com/getsentry/sentry/commit/c9ca2a397e16214bd8c835de31b4b134aa9da907#diff-377ee8f8c07b5ab475d104601a563c9c885289763446a5a00c210612bbb0f727R58 causing the types to widen to `[key: string]: any`